gst/droidcodec/gstdroidvdec.c

```diff
@@ -908,6 +908,19 @@ gst_droidvdec_configure_state (GstVideoDecoder * decoder, guint width,
   return FALSE;
 }
 
+static inline void
+codec_destroy_fn (DroidMediaCodec * codec)
+{
+  if (!codec)
+    return;
+
+  droid_media_codec_stop (codec);
+  droid_media_codec_drain (codec);
+  droid_media_codec_destroy (codec);
+  return;
+}
+
+
```

Comment:
Extra empty line. I think the convention in this file is to have one empty line between functions